src/utils.rs

```diff
@@ -306,9 +306,9 @@ pub(crate) mod testing {
         let mut seeder = OsRng;
         let seed = seeder.gen();
         eprintln!(
-            "Test Failed. To recreate the randomness used, use init_testing_with_seed() with the following seed:"
+            "To re-run test with the same randomness, use init_testing_with_seed() with the following seed:"
```

Comment:
note: Sometimes this gets printed (e.g. when running tests in VSCode) even when the test didn't fail. The old wording was confusing to me so I modified it.

Reply:
Same thing was happening for me on IntelliJ! Thanks for fixing this :pray: